tests/tests_default.yml

```diff
@@ -2,6 +2,5 @@
 ---
 - name: Ensure that the role runs with default parameters
   hosts: all
-
   roles:
-    - linux-system-roles.template
+    - linux-system-roles.mssql
```

Comment:
This will cause an error?  I guess we should use `block/rescue` here and check if the role failed with the `eula not accepted` error

Reply:
Done